packages/rich-text/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+	"presets": ["@wordpress/babel-preset-default"],
+	"plugins": [
+		[ "@babel/plugin-transform-react-jsx", {
+            "pragma": "createElement",
+            "pragmaFrag": "Fragment"
```

Comment:
Is this correct? I didn't see it in other Babel config files, but I got an error when it didn't include this.